db/migrate/20210714215744_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
```

Comment:
The `User` model has been added with an `email` field that is indexed but not unique at the database level. This could lead to potential duplicate records. Consider adding a unique index to the `email` field to enforce uniqueness at the database level.